server/base/src/main/java/org/apache/accumulo/server/util/MetadataTableUtil.java

```diff
@@ -278,15 +278,12 @@ public static void addDeleteEntries(KeyExtent extent, Set<FileRef> datafilesToDe
     // TODO could use batch writer,would need to handle failure and retry like update does -
     // ACCUMULO-1294
     for (FileRef pathToRemove : datafilesToDelete) {
-      update(context,
-          ServerAmpleImpl.createDeleteMutation(context, tableId, pathToRemove.path().toString()),
-          extent);
+      update(context, ServerAmpleImpl.createDeleteMutation(pathToRemove.path().toString()), extent);
```

Comment:
An impl method could take the more precise type, rather than the `toString()` version, without affecting the API.